tests/OpenCFP/Http/Controller/Admin/SpeakersControllerTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace OpenCFP\Tests\Http\Controller\Admin;
+
+use Mockery as m;
+use OpenCFP\Application;
+use OpenCFP\Environment;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorage;
+
+class TalksControllerTest extends \PHPUnit_Framework_TestCase
```

Comment:
`TalksControllerTest` should be `SpeakersControllerTest`. I'll be merging this in the next couple of hours or so. If you can't get to changing this, don't worry about it.